lib/rules/function-no-unknown/README.md

```diff
@@ -35,3 +35,46 @@ a { transform: scale(1); }
 ```css
 a { transform: --custom-function(1); }
 ```
+
+## Optional secondary options
+
+### `ignoreFunctions: ["/regex/", /regex/, "non-regex"]`
+
+Ignore the specified functions.
+
+For example, with `true`.
+
+Given:
+
+```json
+["theme", "/^foo-/"]
+```
+
+The following patterns are considered problems:
+
+<!-- prettier-ignore -->
+```css
+a { transform: unknown(1); }
+```
+
+<!-- prettier-ignore -->
+```css
+a { transform: THEME(1); }
+```
+
+<!-- prettier-ignore -->
+```css
+a { transform: foo(1); }
+```
+
```

Comment:
Let's remove this section.

For `ignore` options we usually just show the rule becoming more permissive in the docs.